src/app/blog/[slug]/components/SharePost.tsx

```diff
@@ -0,0 +1,43 @@
+'use client'
+
+import { clsx } from 'clsx'
+
+import { BASE_URL } from '@/constants/siteMetadata'
+
+import { Icon } from '@/components/Icon'
+import { touchTarget } from '@/components/Social'
+
+import { getSharePostSocialIcons } from '../utils/getSharePostSocialIcons'
+
+type SharePostProps = {
+  title: string
+  postTitle: string
+}
+
+export function SharePost({ title, postTitle }: SharePostProps) {
+  const socialMediaIcons = getSharePostSocialIcons(BASE_URL, postTitle)
+
+  return (
+    <div className="space-y-3 font-bold">
+      <p className="text-brand-300">{title}</p>
+      <ul className="flex w-full gap-6">
```

Comment:
Might want to look into the touchTarget classes / offset.

![Uploading CleanShot 2024-10-22 at 16.08.35@2x.png…]()